README_sqlite_dbs.md

```diff
@@ -25,13 +33,30 @@ more indexes don't always help performance and such performance issues
 need to be analyzed on a case-by-case basis (see
 https://sqlite.org/lang_explain.html).
 
+
+Date fields
+-----------
+
 All the dates are in "%Y-%m-%d" format so that they are automatically
 compatible with SQLite's date functions
 (https://sqlite.org/lang_datefunc.html), Unix core utilities, and
 typical programming language libraries.
 
 
+DB naming convention
+-----------------
+
+The EMR DBs are named following the convention <db_name> "." <data_date>
```

Comment:
I know I said that I prefer BNF-like syntax (and I do), but angle brackets don't work well with Markdown. (All things that look like XML are interpreted as HTML or stripped.) Thus there are 2 options: change the syntax or make the whole thing verbatim (using back ticks or indentation) to quote the angle brackets. The second option would be the easiest.

Also, you could make a single syntax definition for the whole naming convention, including the multiple tags. For example,

    <filename> ::= <db-name> "." <data-date> ("." <tag> ("-" <tag>)*)? ".sqlite3"

Now that I'm thinking about this, it might be better just to provide an example of a name following the convention and then briefly explain it. That would probably be the most clear and efficient. Other characters might be better tag separators, but I'm not sure what. For example,

    mcrf.20180823.snomed-drug_eras.sqlite3
    mcrf.20180823.snomed,drug_eras.sqlite3
    mcrf.20180823.snomed:drug_eras.sqlite3

Out of those, I actually like the colon the best. (In my mind it binds closer than `.` and `,` but not as close as `_` or `-`. However, that isn't necessarily the case for programming language syntaxes, so others will have other opinions based on their experiences and perspectives.)